apps/system/js/app_chrome.js

```diff
@@ -830,7 +830,10 @@
           this.expand();
         }
         this.scrollable.scrollTop = 0;
-        this.pinned = false;
+        // TODO: Check if the new location is pinned
+        // Taking it from the initial value for now
+        var chromeConfig = this.app.config && this.app.config.chrome;
+        this.pinned = chromeConfig && chromeConfig.pinned;
```

Comment:
I think we should probably leave this to bug 1168962 and just always expand browser chrome when the location changes for now. A window should not remain pinned for its entire lifetime.